Pipe.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Pipe : MonoBehaviour
+{
+    PointsUI pointsUi;
+    private void Start() => pointsUi = FindAnyObjectByType<PointsUI>();
+  
+    float pipeDistance = 7f;
+
+    private void OnTriggerEnter2D(Collider2D collision)
+    {
+        pointsUi.AddPoint();
+        StartCoroutine(PreparePipe());
+
+    }
+
+    IEnumerator PreparePipe()
+    {
+        yield return new WaitForSeconds(1f);
+        float height = Random.Range(0.8f, -3f);
```

Comment:
Wartości w kodzie takie jak: `7f, 0.8f, -3f` to tak zwane **"magic numbers"**, staraj się ich unikać. Tutaj masz artykuł o tym:
https://refactoring.guru/replace-magic-number-with-symbolic-constant